gradle.properties

```
@@ -4,4 +4,4 @@ version=1.0.0-SNAPSHOT
 
 releasePluginVersion=2.8.0
 ballerinaGradlePluginVersion=2.2.4
```

Update your Ballerina plugin to 2.3.0

```suggestion
ballerinaGradlePluginVersion=2.3.0
```